airflow/ui/src/theme.ts

```diff
@@ -67,8 +72,16 @@ const theme = extendTheme({
   },
   semanticTokens: {
     colors: {
-      "subtle-bg": { _dark: "gray.900", _light: "blue.50" },
-      "subtle-text": { _dark: "blue.500", _light: "blue.600" },
+      /* eslint-disable perfectionist/sort-objects */
+      contrast: { _dark: "blue.200", _light: "blue.600" },
+      focusRing: "blue.500",
+      fg: { _dark: "blue.600", _light: "blue.400" },
+      emphasized: { _dark: "blue.700", _light: "blue.300" },
+      solid: { _dark: "blue.800", _light: "blue.200" },
+      muted: { _dark: "blue.900", _light: "blue.100" },
```

Comment:
Should these token names be more specific, e.g. bgMuted? Id expect this color to be different for something like muted text (e.g. bodyTextMuted).

Reply:
These are the color names in chakra v3. So I wanted to practice using them. Which `fg` and `emphasized` are the colors for foreground elements like text, borders, icons. This is a bit exploratory too. I might need to adjust it later.